tests/testthat/test-filter.R

```diff
@@ -126,3 +128,63 @@ test_that("scale log weights copes with NaN", {
   expect_equal(res$weights[[3]], 0)
   expect_equal(res, scale_log_weights(w))
 })
+
+
+## This is badly wrong: we've written into some incorrect memory and
+## we never aligned correctly with the steps! Baby steps.
```

Comment:
But fixed now? Or where is this bad write happening?

Reply:
oops, my tests are always littered with notes to myself like this when refactoring...